test-types/describe.test.ts

```diff
@@ -0,0 +1,95 @@
+/**
+ * @type ./empty.d.ts
+ */
+
+import {expectType} from 'tsd';
+
+expectType<void>(describe(0, () => {}));
+expectType<void>(describe('name', () => {}));
+expectType<void>(
+  describe(
+    () => {},
+    () => {},
+  ),
+);
+expectType<void>(describe({name: 'name'}, () => {}));
+
+expectType<void>(describe.only(0, () => {}));
```

Comment:
most of the lint errors should probably be added ignored via `.eslintrc.js` since we intentionally use APIs here that we would discourage in all other places

Reply:
Done